src/LandPlanner.cpp

```diff
@@ -104,16 +113,31 @@ bool LandPlanner::getTargetMotionPoint(const ros::Time& time,
         return false;
     }
 
+    actual_height_ = transform.transform.translation.z;
```

Comment:
The height doesn't need to be stored off to a class member since no calculations depends on knowing the height from the last iteration. We can get rid of this and just use "transform.transform.translation.z" everywhere.